Automattic/simplenote-android
# Update Tablet Divider

### Fix
Update the divider between the note list and the note editor/preview from a shadow to a line.  See the screenshots below for illustration.

Before|After
-|-
![update_tablet_divider_before_light](https://user-images.githubusercontent.com/3827611/67351338-8157be00-f50a-11e9-85f7-ff49919353e9.png)|![update_tablet_divider_after_light](https://user-images.githubusercontent.com/3827611/67351347-887ecc00-f50a-11e9-98f4-9b5d423a937f.png)
![update_tablet_divider_before_dark](https://user-images.githubusercontent.com/3827611/67351353-8a488f80-f50a-11e9-8162-33fe46d85048.png)|![update_tablet_divider_after_dark](https://user-images.githubusercontent.com/3827611/67351363-8e74ad00-f50a-11e9-9ced-9fde53203791.png)

### Test
The divider is only visible with the two-pane layout on large screen devices (e.g. tablet). When performing the test steps below, be sure to use a large screen device.
1. Rotate device into landscape orientation.
2. Notice divider line is shown.
3. Tap any note in list.
4. Notice divider line is shown.
5. Rotate device into portrait orientation.
6. Notice divider line is hidden.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
